rigs/kenwood/k3.c

```diff
@@ -1066,28 +1066,39 @@ int k3_set_mode(RIG *rig, vfo_t vfo, rmode_t mode, pbwidth_t width)
 
     switch (mode)
     {
-    case RIG_MODE_PKTLSB:
-        mode = RIG_MODE_RTTY;
+        case RIG_MODE_PKTLSB:
+        mode = RIG_MODE_RTTYR;
```

Comment:
For some reason this had to be RTTYR for rigctl/set_mode PKTLSB 0 to send the rig to DATA REV submode data A (DT0)